failure_test.go

```diff
@@ -260,6 +188,7 @@ func (e *ep) Setup() {
 	e.valid = true
 	e.resp = testBlock
 	e.server = httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(time.Millisecond * 20)
```

Comment:
why is this needed? / why are responses delayed?

Reply:
Just want to introduce some form of delay in the tests as this is the "L1 peer that sends us blocks".

Reply:
doing so masks race conditions / makes tests artificially slow - we ideally shouldn't need it - add TODO to further clean up tests